### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by Mre55**
### Leaderboard: hit the API

- Create a new game with the name of my choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,30 +1,50 @@
...
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const gameResult = await response.json();
+  return gameResult;
+}
+
+async function fetchDataFromAPI() {
+  const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`);
+  const getResult = await response.json();
```
Comment (Dipeshtwis) [inline src/index.js]: - [x] Ensure to use of arrow functions instead of the `function` keyword as this is a project requirement to use ES6 syntax.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Mre55, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The app is working as expected.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] I would recommend including the steps to run the project like `npm i` to install the packages and `npm start` to run the server. Currently, opening `index.html` directly won't work in sending and getting the score. Please update the command.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Mre55) [reply src/index.js]: Resolved

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Mre55,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**